packages/core/src/dsl/DeploymentBuilder.ts

```diff
@@ -567,10 +573,26 @@ export class DeploymentBuilder implements IDeploymentBuilder {
     return this.idCounter++;
   }
 
+  private static _captureCallPoint(
+    callPoints: CallPoints,
+    // eslint-disable-next-line @typescript-eslint/ban-types
+    f: Function,
+    vertexId: number
+  ) {
+    const potentialValidationError = new IgnitionValidationError("");
+    potentialValidationError.resetStackFrom(f);
+    callPoints[vertexId] = potentialValidationError;
+  }
+
   private static _addVertex(
     graph: DeploymentGraph,
+    callPoints: CallPoints,
+    // eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
What is the error being suppressed here? Is it because `f` is typed as a generic `Function` and not something more descriptive?

Reply:
will `f: () => any` fix the error?

Reply:
That doesn't quite work due to ts generics. I reworked to be explicit about the public functions should be passed to add vertex and then elided the type to any to pass it to the underlying function.

Take a look at the commit, or if you think the original approach is simpler, let me know.